### DISCUSSION TITLE
Is it possible to minimize budget with constant volume?

### DISCUSSION THREAD

**Discussion by thipokKub** [google/lightweight_mmm#169] [Ideas]
From the [docs](https://lightweight-mmm.readthedocs.io/en/latest/api.html#optimize-media), lightweight_mmm.optimize_media.find_optimal_budget optimize for maximum volume given a fixed budget. However there is no option to minimize budget for a fixed volume. It might be possible to perform binary search on budget such that the expected desire volume is obtained (if convex), but it would be nicer to have them natively implemented

**Comment by statm3n** [comment]
Thanks for raising this idea! We will keep evaluating the needs of this minimize budget for a fixed volume scenario.
